src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -314,7 +329,19 @@ export class MeetingManager implements AudioVideoObserver {
 
   async listAndSelectDevices(): Promise<void> {
     await this.updateDeviceLists();
```

Comment:
Keep the original design. Since we still need to call all `init*Devices()` function in `DeviceProvider`s when user calls `meetingManager.join()`. It's okay that we update all the device lists in any cases.